py/server/deephaven/agg.py

```diff
@@ -269,17 +278,29 @@ def std(cols: Union[str, List[str]] = None) -> Aggregation:
     return Aggregation(j_agg_spec=_JAggSpec.std(), cols=cols)
 
 
-def unique(cols: Union[str, List[str]] = None) -> Aggregation:
-    """Create a Unique aggregation.
+def unique(cols: Union[str, List[str]] = None, include_nulls: bool = False, null_sentinel: Any = None) -> Aggregation:
+    """Create a Unique aggregation which computes the single unique value within an aggregation group for each of
+    the given columns. If all values in a column are null, or if there is more than one distinct value in a column, the
+    result is null.
 
     Args:
         cols (Union[str, List[str]]): the column(s) to aggregate on, can be renaming expressions, i.e. "new_col = col";
             default is None, only valid when used in Table agg_all_by operation
+        include_nulls (bool): whether null is treated as a value for the purpose of determining if the values in the
+            aggregation group are unique, default is False.
+        null_sentinel (Any): the non-null sentinel value to substitute null when include_nulls is True, default is None
```

Comment:
I'm not sure None makes sense here. Will that result in us storing `null`? Are you unwrapping `PyObject`s?

Reply:
1. null_sentinel is only applied when include_nulls is True
2. when include_nulls is True, we check to make sure null_sentinel be non-None

Reply:
I think there's some confusion here. The "non-unique sentinel" is what you should be wrapping. Not the "null sentinel". Look at `AggSpecUnique`.